### ISSUE TITLE
Pi 3, Raspbian Stretch Lite, go1.8.3 armv6l, ELI5?

### ISSUE THREAD

**Issue by TheButterZone** [golang/go#21821]
```
pi@pi:~ $ dir
go  go1.8.3.linux-armv6l.tar.gz
pi@pi:~ $ cd /usr/local/go/
pi@pi:/usr/local/go $ dir
api	 blog		  doc	       LICENSE	pkg	    src
AUTHORS  CONTRIBUTING.md  favicon.ico  misc	README.md   test
bin	 CONTRIBUTORS	  lib	       PATENTS	robots.txt  VERSION
pi@pi:/usr/local/go $ 
pi@pi:/usr/local/go $ sudo nano VERSION
go1.8.3
pi@pi:/usr/local/go $ go version
-bash: go: command not found
pi@pi:/usr/local/go $ cd
pi@pi:~ $ go version
-bash: go: command not found
pi@pi:~ $ sudo reboot
...
Linux pi 4.9.41-v7+ #1023 SMP Tue Aug 8 16:00:15 BST 2017 armv7l
pi@pi:~ $ go version
-bash: go: command not found
pi@pi:~ $ go get github.com/OpenBazaar/openbazaar-go
-bash: go: command not found
```

**Comment by as** [comment]
Please don't ask questions on the issue tracker.
https://github.com/golang/go/wiki/Questions

The wiki covers building from source, it's probably faster to find the answer there
https://github.com/golang/go/wiki

**Comment by TheButterZone** [comment]
It's an issue because installation instructions don't work & it seems like the Pi 3 is deprecated because of it.

https://github.com/golang/go/issues/15433

**Comment by as** [comment]
Your session doesn't suggest that an issue exists.

```
pi@pi:/usr/local/go $ go version
-bash: go: command not found
```

- The go executable is in the `go/bin` subdirectory. You are trying to execute it one directory level above where it actually is. Not working, as intended.
- It's also not in your $PATH, so you'll need to run `./go` and not `go`
- You also need to ensure it has execute permissions
- You have an armv6 tarball

If you have further questions, please refer to the links above or ask the question on golang-nuts

**Comment by TheButterZone** [comment]
golang-nuts is not a channel. As far as the other stuff...
`pi@pi:~ $ cd /usr/local/go/bin
pi@pi:/usr/local/go/bin $ go version
-bash: go: command not found
pi@pi:/usr/local/go/bin $ ./go version
go version go1.8.3 linux/arm
pi@pi:/usr/local/go/bin $ ./go get github.com/OpenBazaar/openbazaar-go
go install runtime/internal/atomic: open /usr/local/go/pkg/linux_arm/runtime/internal/atomic.a: permission denied
pi@pi:/usr/local/go/bin $ cd
pi@pi:~ $ echo "export GOPATH=$HOME/go" >> .profile
pi@pi:~ $ echo "export PATH=$PATH:/usr/local/go/bin" >> .profile
pi@pi:~ $ source ~/.profile
pi@pi:~ $ go get github.com/OpenBazaar/openbazaar-go
go install runtime/internal/atomic: open /usr/local/go/pkg/linux_arm/runtime/internal/atomic.a: permission denied
pi@pi:~ $ 
`

I'm still 5 years old & you are adding a 0 to my age.

**Comment by dsnet** [comment]
Thanks for trying to learn go, but the link at https://github.com/golang/go/wiki/Questions provides several ways to ask questions to the community. Some of them like Gophers Slack and the IRC are a channel-like medium for more real-time help.

**Comment by TheButterZone** [comment]
https://github.com/golang/go/issues/15433 seems like a related issue, and no attempt was made to redirect it to other mediums.

**Comment by davecheney** [comment]
You need to add /usr/local/go/bin your $PATH as detailed here

https://golang.org/doc/install#tarball

I am locking this conversation now, if you have further problems please follow the links to appropriate places to ask installation questions provided above.
